app/services/hyrax/thumbnail_path_service.rb

```diff
@@ -22,6 +22,11 @@ def call(object)
 
       private
 
+      def audio?(thumb)
+        service = thumb.respond_to?(:audio?) ? thumb : Hyrax::FileSetTypeService.new(file_set: thumb)
```

Comment:
no longer adding `Hydra::Works::MimeTypes` to generated classes. a partial re-implementation is provided as `Hyrax::FileSetTypeService`